platform/pulpcore/app/viewsets/repository.py

```diff
@@ -140,6 +140,19 @@ class PublisherViewSet(NamedModelViewSet):
     queryset = Publisher.objects.all()
     filter_class = PublisherFilter
 
+    def update(self, request, *args, **kwargs):
```

Comment:
Same for here, we should know all positional and keyword arguments since we are defining the API. Please convert to named positional and keyword arguments.